client/store/taxes/selectors.js

```diff
@@ -0,0 +1,43 @@
+/** @format */
+/**
+ * External dependencies
+ */
+import { get } from 'lodash';
+import { select } from '@wordpress/data';
+/**
+ * Internal dependencies
+ */
+import { ERROR } from 'store/constants';
+import { getJsonString } from 'store/utils';
+/**
+ * Returns taxes report details for a specific report query.
+ *
+ * @param  {Object} state  Current state
+ * @param  {Object} query  Report query paremters
+ * @return {Object}        Report details
+ */
+function getTaxes( state, query = {} ) {
+	return get( state, [ 'taxes', getJsonString( query ) ], [] );
```

Comment:
Should the default return here be an object `{}` not an array?

Reply:
Done